ctapipe/reco/reconstructor.py

```diff
@@ -141,6 +141,18 @@ def read(cls, path, parent=None, subarray=None, **kwargs):
         Provenance().add_input_file(path, role="reconstructor")
         return instance
 
+    def set_n_jobs(self, n_jobs):
+        """
+        Set n_jobs if applicable for the reconsructor.
+        This is not just a traits option of the class, because
+        for example in the case of the SKLearnReconstructors you need
+        to set the property of all of the sklearn models for it to be
+        applied in the fit and predict steps.
+        """
+        self.log.warning(
```

Comment:
No warning please for thing that are expected.

Reply:
Are they expected?
This should only happen if a user specifically wants to overwrite a value and they expect that to work and not be ignored.

Reply:
I'd expect that we add this option to all tools or via global configuration. I.e. the number of cores ctapipe as a whole is allowed to use in parallel.

Anything that can only use one core should just ignore it.

Reply:
Hm, if we extend this to every step of ctapipe via a global setting I agree.
I was more concerned about the specific usage of `ctapipe-apply-models` and was assuming
the lower datalevels would not be parallelized on the level of a single file anyway.